toolkit/tools/pkg/imagecustomizerlib/liveosisobuilder.go

```diff
@@ -227,18 +239,26 @@ func (b *LiveOSIsoBuilder) stageIsoMakerInitrdArtifacts(writeableRootfsDir, isoM
 		return fmt.Errorf("failed to create %s\n%w", targetBootloadersDir, err)
 	}
 
-	sourceBoot64EfiPath := b.artifacts.bootx64EfiPath
-	targetBoot64EfiPath := filepath.Join(targetBootloadersDir, bootx64Binary)
-	err = file.Copy(sourceBoot64EfiPath, targetBoot64EfiPath)
+	sourceBootEfiPath := b.artifacts.bootx64EfiPath
+	targetBootEfiPath := filepath.Join(targetBootloadersDir, bootx64Binary)
+	if runtime.GOARCH == "arm64" {
```

Comment:
This way requires reading through the logic to see that the initial assignments might be changed later. Using a switch statement for `runtime.GOARCH` could make the architecture dependent assignment explicit.